src/qt/pivx/forms/addresseswidget.ui

```diff
@@ -229,7 +229,7 @@
                  <string notr="true"/>
                 </property>
                 <property name="text">
-                 <string>No active Master Node yet</string>
+                 <string>No active Masternode yet</string>
```

Comment:
This is actually a dummy string (it is set in the `.cpp` file to it's visible value). This file here doesn't even have anything to do with MNs.

Rather than have a duplicate string here, would change this line to read
```html
<string notr="true">N/A</string>
```